pkg/odo/cli/component/devfile.go

```diff
@@ -0,0 +1,78 @@
+package component
+
+import (
+	"fmt"
+
+	"github.com/openshift/odo/pkg/devfile"
+	"github.com/openshift/odo/pkg/odo/cli/project"
+	"github.com/openshift/odo/pkg/odo/genericclioptions"
+	"github.com/spf13/cobra"
+	ktemplates "k8s.io/kubernetes/pkg/kubectl/cmd/templates"
+)
+
+// TODO update examples
+var pushDevFileExample = ktemplates.Examples(`  # TODO
```

Comment:
Please add a note here, or comment at the beginning of this file explaining that currently, this is just for testing purposes.

Reply:
Done! added a note.